vita3k/kernel/src/thread/sync_primitives.cpp

```diff
@@ -728,7 +746,11 @@ int eventflag_set(KernelState &kernel, const char *export_name, SceUID thread_id
     }
 
     const std::lock_guard<std::mutex> event_lock(event->mutex);
-    event->flags |= flags;
+    if (do_clear) {
+        event->flags &= !flags;
```

Comment:
```suggestion
        event->flags &= ~flags;
```
use bitwise not?

Reply:
Shouldn't this be just event->flags &= flags?

Reply:
1whatleytay Yes, tnx
sunho No. This is "The bits to clean" https://docs.vitasdk.org/group__SceThreadMgrUser.html#gab576197fc9afd63693c7661d1f46a257

Reply:
The game specify 0 if it wants to clear that bit. Can you join the discord server?